geopandas/base.py

```diff
@@ -6078,7 +6112,9 @@ def cx(self):
         """
         return _CoordinateIndexer(self)
 
-    def get_coordinates(self, include_z=False, ignore_index=False, index_parts=False):
+    def get_coordinates(
+        self, include_z=False, ignore_index=False, index_parts=False, include_m=False
```

Comment:
The order here is a bit unfortunate but it is the same in shapely. Over there it triggered deprecation of positional args though. Maybe something to consider as well.

Reply:
I think we probably want to follow https://github.com/shapely/shapely/pull/2234/files, or at very least make include_m keyword only.